src/woo_publications/api/constants.py

```diff
@@ -5,3 +5,9 @@
 class PermissionOptions(models.TextChoices):
     read = "read", _("Read")
     write = "write", _("Write")
+
+
+class PublicationStatusOptions(models.TextChoices):
```

Comment:
since this is relevant only to the `Publication` and `Document` models, which are both (deliberately :wink: ) defined in the `publications` app, this enum must also be defined in that app. Can you please move it?